templates/note.php

```diff
@@ -1,4 +1,2 @@
-<div id="app-navigation-toggle" class="icon-menu" style="display:none;"></div>
 	<pre editor notes-timeout-change="save()" notes-autofocus class="editor" ng-class="{saving: isSaving()}">{{note.content}}</pre>
```

Comment:
This change fixes #369. I'm very interested to see it merged @DeepDiver1975 @jnweiger